src/ResourceManager/Sql/Commands.Sql/Database/Cmdlet/NewAzureSqlDatabase.cs

```diff
@@ -101,6 +100,11 @@ public class NewAzureSqlDatabase : AzureSqlDatabaseCmdletBase
         [Alias("Tag")]
         public Hashtable Tags { get; set; }
 
+        [Parameter(Mandatory = false,
+            HelpMessage = "The name of the sample schema to apply when creating this database.")]
+        [ValidateNotNullOrEmpty]
+        public string SampleName { get; set; }
```

Comment:
This parameter is used to create a database populated with some sample tables, data, etc. The only allowed sample name for now is `AdventureWorksLT` which is one of the sample db's we publish for SQL server. See https://msftdbprodsamples.codeplex.com/ . However in theory there could be other samples in the future.